pkg/kubelet/kuberuntime/kuberuntime_container_linux_test.go

```diff
@@ -735,6 +736,7 @@ func TestGenerateLinuxContainerResources(t *testing.T) {
 			v1.ResourceList{v1.ResourceCPU: resource.MustParse("250m"), v1.ResourceMemory: resource.MustParse("500Mi")},
 			[]v1.ContainerStatus{},
 			&runtimeapi.LinuxContainerResources{CpuShares: 256, MemoryLimitInBytes: 524288000, OomScoreAdj: -997},
+			cgroupV1,
```

Comment:
code change LGTM. But we are placing the cgroupv1 into the maintanence mode: https://github.com/kubernetes/enhancements/issues/4569

This would mean no new features in cgroupv1, I would expect we will cut the support for cgroupv1 in this KEP.

cc: @harche